packages/openapi3/src/cli/actions/convert/emitters/emit-main.ts

```diff
@@ -0,0 +1,27 @@
+import { formatTypeSpec } from "@typespec/compiler";
+import { generateModel } from "../generators/generate-model.js";
+import { generateOperation } from "../generators/generate-operation.js";
+import { generateServiceInformation } from "../generators/generate-service-info.js";
+import { TypeSpecProgram } from "../interfaces.js";
+
+export async function emitMain(program: TypeSpecProgram): Promise<string> {
```

Comment:
is this file sutrcuture and naming from the autorest plugin? Feel like we could simpliy it a little bit, not sure there is a difference betteween emitters/generators

Reply:
It is - and agreed that `emit-main` is equivalent to the generators. Will update.